cdc/capture/capture.go

```diff
@@ -260,6 +260,17 @@ func (c *Capture) Run(ctx context.Context) error {
 			log.Info("capture recovered", zap.String("captureID", c.info.ID))
 			continue
 		}
+		select {
+		case <-c.session.Done():
+			log.Warn("capture encountered error while session is gone. Retrying later.",
```

Comment:
1. Should it be written as "capture encountered error while session is **done**"?
2. Can we throw the session done [here](https://github.com/pingcap/tiflow/blob/d141ee67f015377cfc288e5ea68c101614eaed16/cdc/capture/capture.go#L471) and handle it uniformly.

Reply:
1. **gone** feels equally okay to me.
2. This error might not be the immediate cause of the problem, so we need to consider all types of errors that can be caused by the PD cluster being unavailable.